python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -357,6 +360,42 @@ def test_smoothing_factor_with_0_replica():
     assert new_num_replicas == 1
 
 
+def test_smoothing_factor_scale_down_to_0_replica():
+    """Test that a deployment scales down to 0 for non-default smoothing factors."""
+
+    # Smoothing factor = 10
```

Comment:
The comments in the test are not useful. 
`# Smoothing factor = 10` is just describing `smoothing_factor=10,`. The same for the following comments. it would be great to provide more concrete scenario description. Or we can just delete it.

Reply:
I've rewritten the comments to elaborate on the two things I'm testing!